server/graphql/v2/interface/Account.ts

```diff
@@ -674,6 +677,43 @@ const accountFieldsDefinition = () => ({
     type: new GraphQLNonNull(GraphQLAccountPermissions),
     description: 'Logged-in user permissions on an account',
   },
+  feed: {
+    type: new GraphQLList(GraphQLActivity),
+    describe: 'Get the activity feed for this account',
+    args: {
+      untilId: {
+        type: GraphQLString,
+        description: 'Only returns activities before ID',
+      },
```

Comment:
I prefer timestamp based interfaces.

And generally we're trying to not expose internal autoincrement ids.

Reply:
I'll refactor this, I agree using timestamp is better.